src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -130,17 +157,58 @@ const SuggestedOwners = createReactClass({
     );
   },
 
+  renderOwner(owner) {
+    let {rule} = this.state;
+    return (
+      <span
+        key={`${owner.id}:${owner.type}`}
+        className="avatar-grid-item tip"
+        onClick={() => this.assignToActor(owner)}
+        title={ReactDOMServer.renderToStaticMarkup(
```

Comment:
😬

Reply:
is this that bad? better than building a string, right?

Reply:
it is gross that we send tooltips through bootstrap js